xdocs/stylesheets/site_printable.vsl

```diff
@@ -239,6 +239,10 @@ No
 <a href="https://github.com/apache/jmeter/pull/$id.getText()">Pull request #$id.getText()</a>##
 #end
 
+#macro ( issue $id)
```

Comment:
If we start adding `<issue>` tags, we should add support for it in `website-style.xsl`, too.

Reply:
I have seen it now, I thought it would be in this PR.